TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/search/LocationSearchViewModel.kt

```diff
@@ -310,7 +316,7 @@ class LocationSearchViewModel @Inject constructor(
 
         onQueryTextChangeEventThrottle.debounce(500, TimeUnit.MILLISECONDS)
             .subscribe({
-                this.searchResults(it)
+                searchResults(if (it == "Current Location") it else "")
```

Comment:
Hi @sg-jsonjuliane, could we consider assigning `Current Location` to a variable, or utilizing a Local resource for it? 
Additionally, we have the `SearchResultItemSource` available, which allows access to `CurrentLocation`. Given its relevance to search results, this enum class seems like an ideal match.

Reply:
@MichaelReyes updated the PR. Thank you!